### PR TITLE
new tag: filter

### PR DESCRIPTION
**Pull request by thehairy**
This PR adds the filter tag.

### CODE REVIEW

File: tags/tags.toml [hunk 1]
```diff
@@ -803,3 +803,13 @@ Version 13 is still in development, the branch moves quickly.
 • [Preview update guide](<https://deploy-preview-680--discordjs-guide.netlify.app/additional-info/changes-in-v13.html>) (use `CTRL` + `F` to search for the old method or property)
 *\\🚧 This is still a draft and may not be up to date at all times*
 """
+
+[filter]
+keywords = ["filter"]
+content = """
+In v13 the Collector filter has moved into the options object:
```
Comment (iCrawl) [inline tags/tags.toml:810]:
```suggestion
The filter for collectors has moved into the options:
```

**Comment by anandre** [comment]
Wouldn't this fall under using the guide interaction to direct them to the v13 changes page?

**Comment by thehairy** [comment]
> Wouldn't this fall under using the guide interaction to direct them to the v13 changes page?

I don't see a point to redirect them to a whole guide just for this small change.
You can't tell me that you read a guide and didn't miss a single thing.

**Review by iCrawl — APPROVED** [review]

**Comment by anandre** [comment]
> I don't see a point to redirect them to a whole guide just for this small change.
> You can't tell me that you read a guide and didn't miss a single thing.

I was of the same mind for the v12changes tag, which was a link to the changes page and told users to Ctrl + F the old method.  I don't see why that wouldn't apply here, and Souji has already removed most if not all of the "read the guide" tags.

**Review by SpaceEEC — APPROVED** [review]

**Review by almostSouji — APPROVED** [review]
After trying to use the guide command for some more pinpoint explanations, it might make sense to re-introduce some  of the quickfire-answer tags

**Pull request merged**